src/Maestro/tests/Maestro.ScenarioTests/ScenarioTests_MergePolicies.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Microsoft.Azure.KeyVault.Models;
+using Microsoft.DotNet.Maestro.Client.Models;
+using NUnit.Framework;
+
+
+namespace Maestro.ScenarioTests
+{
+    [TestFixture]
+    [Category("PostDeployment")]
+    public class ScenarioTests_MergePolicies : MaestroScenarioTestBase
+    {
+        private TestParameters _parameters;
+        private Random _random = new Random();
```

Comment:
Random is really scary in tests, because it means they can randomly fail.  What sorts of stuff are we trying to accomplish with random, and is there some other way to accomplish it?

Reply:
Create a random ID for the channel name. It was the way it was used in the powershell script (by using a random function).